File: Codigo/SistemaVeterinario/logica/Propietario.cs
namespace SistemaVeterinario.logica
{
    public class Propietario : Persona
    {
        public int intCodigo { get; set; }

        public Propietario(int intCodigo, string strNombre, string strTelefono)
            : base(strNombre, strTelefono)
        {
            this.intCodigo = intCodigo;
        }

        public void mostrarPropietario()
        {
            Console.WriteLine("Codigo: " + intCodigo);
            Console.WriteLine("Nombre: " + strNombre);
            Console.WriteLine("Telefono: " + strTelefono);
        }
    }
}

File: SistemaVeterinario/SistemaVeterinario/Propietario.java
public class Propietario extends Persona
{
    int intCodigo;

    public Propietario(int intCodigo, String strNombre, String strTelefono)
    {
        super(strNombre, strTelefono);
        this.intCodigo = intCodigo;
    }

    public void mostrarPropietario()
    {
        System.out.println("Codigo: " + intCodigo);
        System.out.println("Nombre: " + strNombre);
        System.out.println("Telefono: " + strTelefono);
    }
}